server/main.lua

```diff
@@ -1,7 +1,15 @@
 local config = require 'config.server'
 local sharedConfig = require 'config.shared'
 local VEHICLES = exports.qbx_core:GetVehiclesByName()
-local outsideVehicles = {}
+
+local function findPlateOnServer(plate)
+    local vehicles = GetAllVehicles()
```

Comment:
May want run a before/after performance test of this. Wonder if this could be a hotspot.

Reply:
The operation is executed rarely, but some filtering and caching can be used.

Reply:
I recall this native being an issue even being ran only once, but I guess we'll find out during future testing.